src/main/java/duke/command/EventCommand.java

```diff
@@ -0,0 +1,91 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Add event command.
```

Comment:
For header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Add or Adding etc.)

Reply:
Thank you for letting me know! I'll adjust it!